src/unix.rs

```diff
@@ -1,65 +1,40 @@
-extern crate libc;
-use std::os::raw::*;
-use std::os::unix::io::RawFd;
-
+#[cfg(unix)]
 use super::{Height, Width};
 
-#[cfg(target_os = "macos")]
-const TIOCGWINSZ: c_ulong = 0x40087468;
-#[cfg(all(target_env = "musl", not(target_os = "macos")))]
-const TIOCGWINSZ: c_int = 0x00005413;
-#[cfg(all(not(target_env = "musl"), not(target_os = "macos")))]
-const TIOCGWINSZ: c_ulong = 0x00005413;
-
-#[derive(Debug)]
-struct WinSize {
-    ws_row: c_ushort,
-    ws_col: c_ushort,
-    ws_xpixel: c_ushort,
-    ws_ypixel: c_ushort,
-}
-
 /// Returns the size of the terminal defaulting to STDOUT, if available.
 ///
 /// If STDOUT is not a tty, returns `None`
 pub fn terminal_size() -> Option<(Width, Height)> {
-    use self::libc::STDOUT_FILENO;
-    terminal_size_using_fd(STDOUT_FILENO)
+    terminal_size_using_fd(libc::STDOUT_FILENO)
 }
 
 /// Returns the size of the terminal using the given file descriptor, if available.
 ///
 /// If the given file descriptor is not a tty, returns `None`
-pub fn terminal_size_using_fd(fd: RawFd) -> Option<(Width, Height)> {
-    use self::libc::ioctl;
-    use self::libc::isatty;
+pub fn terminal_size_using_fd(fd: i32) -> Option<(Width, Height)> {
```

Comment:
What was the motivation to change this type from `RawFD` to `i32`?

Reply:
changed it back.